metadata/editions.asciidoc

```diff
@@ -0,0 +1,13 @@
+== GITenberg metadata: Editions
+
+There is a need to be able to express metadata for multiple editions in one repository. Different editions will share most metadata, but may differ in ISBN, publisher, cover, rights, etc.
+
+To support this need, pandata provides properties the "_edition", "edition_list", edition_note "edition_identifiers", and any other property beginning with "edition_"
```

Comment:
"provides properties the" ? :) maybe meant to be: "provides the following properties: "

Also, should `edition_note` also have quotation marks around it? (Tiny thing, i know... )